.projenrc.ts

```diff
@@ -1,12 +1,193 @@
-import { CdklabsConstructLibrary } from 'cdklabs-projen-project-types';
-const project = new CdklabsConstructLibrary({
-  author: 'AWS',
-  authorAddress: 'aws-cdk-dev@amazon.com',
-  cdkVersion: '2.1.0',
-  defaultReleaseBranch: 'main',
-  devDeps: ['cdklabs-projen-project-types'],
-  name: '@cdklabs/cdk-amazonmq-activemq-configs',
-  projenrcTs: true,
-  release: false,
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import fs from "fs";
+import {
+  CdkLabsMonorepo,
+  TypeScriptWorkspace,
+} from "cdklabs-projen-project-types/lib/yarn";
+import { ReleasableCommits, SampleDir, TextFile } from "projen";
+import { Jsii } from "./projenrc/jsii";
+import { Stability } from "projen/lib/cdk";
+import { SourceUpdate } from "./projenrc/update-source";
+
+const stability = Stability.EXPERIMENTAL;
+
+const project = new CdkLabsMonorepo({
+  defaultReleaseBranch: "main",
+  stability,
+  deps: ["semver"],
+  devDeps: ["cdklabs-projen-project-types", "@types/semver"],
+  name: "@cdklabs/cdk-amazonmq-activemq-configs",
+  github: true,
+  release: true,
+  tsconfigDev: {
+    compilerOptions: {
+      lib: ["ES2021", "ES2021.String"]
+    }
+  }
 });
-project.synth();
\ No newline at end of file
+
+new SourceUpdate(project, {
```

Comment:
My understanding is that this will read from the documentation and create `sources` file. But I don't see one in this intial commit. Is that expected?

Reply:
Yes. The repository was generated with standard presets. We're adopting the same approach as can be found in the [awscdk-service-spec repo](https://github.com/cdklabs/awscdk-service-spec).